src/test/java/com/epam/task/third/comparators/ConeComparatorTest.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.task.third.comparators;
+
+import com.epam.task.third.enteties.Circle;
+import com.epam.task.third.enteties.Cone;
+import com.epam.task.third.enteties.ConeObservable;
+import com.epam.task.third.enteties.Point;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Comparator;
+
+public abstract class ConeComparatorTest {
+    private Comparator <Cone> coneCenterComparator = creatingConeCenterComparatorTest();
+    private static final Point BIGGER_CONE_CENTER = new Point(1.0,1.0,1.0);
```

Comment:
do not try to make universal data , there is always the case when your data won't fit, so create different tests with different test data. do not be lazy